lib/services/workflow-api-service.js

```diff
@@ -155,7 +237,32 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.find({ node: target })
+            .then(function(workflows){
+                return _.filter(workflows, function(wf) {
+                    return wf.active();
```

Comment:
@brianparry I agree, I think the right thing to do might be to change the state to `Constants.Task.States.Running` since it's a pre-existing state? We didn't do it originally because it saved a query operation, but I think it makes sense given how confusing it is.

Ah ok, yeah I remember making some mistakes/inconsistencies with `id` vs. `_id`, so it's possible I didn't resolve all of them.